### PR TITLE
enable CDI on containerd

### PR DESCRIPTION
**Pull request by aojea**
I think this is what we just need for supporting CDI devices in kind

 I still need to test it

### CODE REVIEW

**Comment by klueska** [comment]
Yes, this is all that is required. Though the specification of the dirs is unnecessary, as the ones you provide are the defaults.

See the following for how we have been enabling this in one of our repos via `containerdConfigPatches`:
https://github.com/NVIDIA/k8s-dra-driver/blob/main/demo/clusters/kind/scripts/kind-cluster-config.yaml#L20-L25

**Review by klueska — APPROVED** [review]

File: images/base/files/etc/containerd/config.toml [hunk 1]
```diff
@@ -40,3 +40,6 @@ version = 2
   tolerate_missing_hugepages_controller = true
   # restrict_oom_score_adj needs to be true when running inside UserNS (rootless)
   restrict_oom_score_adj = false
+  # enable CDI https://github.com/cncf-tags/container-device-interface
+  enable_cdi = true
```
Comment (BenTheElder) [inline images/base/files/etc/containerd/config.toml:44]: why is this default false? 👀

Comment (aojea) [reply images/base/files/etc/containerd/config.toml:44]: @estesp @mikebrow can you help us here ? ^^

Comment (aojea) [reply images/base/files/etc/containerd/config.toml:44]: I see, is a 2.0 thing https://github.com/containerd/containerd/commit/c8e8a093ce3dd9231173042e9828a679972f5d6b

Comment (estesp) [reply images/base/files/etc/containerd/config.toml:44]: correct

**Comment by aojea** [comment]
it will be part of 2.0 containerd

**Pull request closed without merging**